forgerock-auth/src/test/java/org/forgerock/android/auth/devicebind/ApplicationPinDeviceAuthenticatorTest.kt

```diff
@@ -124,7 +124,7 @@ class ApplicationPinDeviceAuthenticatorTest {
         authenticator2.prompt(prompt)
         authenticator2.setKey(cryptoKey)
         val status = authenticator2.authenticate(context)
-        assertThat(status).isEqualTo(UnAuthorize())
```

Comment:
Is there a way to use UnAuthorize when the Pin is invalid?
Unreg - The device has not been registered
UnAuth - The device is registered, but user provide the wrong pin to unlock.